pkg/scheduler/flavorassigner/flavorassigner_test.go

```diff
@@ -513,71 +427,15 @@ func TestAssignFlavors(t *testing.T) {
 					Request(corev1.ResourceCPU, "3").
 					Obj(),
 			},
-			clusterQueue: cache.ClusterQueue{
-				ResourceGroups: []cache.ResourceGroup{
-					{
-						CoveredResources: sets.New(corev1.ResourceCPU),
-						Flavors: []cache.FlavorQuotas{
-							{
-								Name: "tainted",
-								Resources: map[corev1.ResourceName]*cache.ResourceQuota{
-									corev1.ResourceCPU: {Nominal: 4000},
-								},
-							},
-							{
-								Name: "two",
-								Resources: map[corev1.ResourceName]*cache.ResourceQuota{
-									corev1.ResourceCPU: {Nominal: 4000},
-								},
-							},
-						},
-					},
-				},
-			},
-			wantRepMode: Fit,
-			wantAssignment: Assignment{
-				PodSets: []PodSetAssignment{{
-					Name: "main",
-					Flavors: ResourceAssignment{
-						corev1.ResourceCPU: {Name: "two", Mode: Fit, TriedFlavorIdx: -1},
-					},
-					Requests: corev1.ResourceList{
-						corev1.ResourceCPU: resource.MustParse("3000m"),
-					},
-					Count: 1,
-				}},
-				Usage: resources.FlavorResourceQuantitiesFlat{
-					{Flavor: "two", Resource: corev1.ResourceCPU}: 3000,
-				}.Unflatten(),
-			},
-		},
-		"multiple flavors, skip missing ResourceFlavor": {
```

Comment:
I'm wondering if there could be some race conditions if scheduler tries to use them before they are marked inactive? Maybe it is safer to keep the test in some form?